functions/onprem/orborus/orborus.go

```diff
@@ -154,6 +154,35 @@ func init() {
 
 }
 
+func getDockerClient(ctx context.Context) (*dockerclient.Client, error) {
+	cli, err := dockerclient.NewEnvClient()
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = cli.Info(ctx)
+	if err == nil {
+		return cli, nil
+	}
+
+	if strings.Contains(err.Error(), "Minimum supported API version is") {
+		re := regexp.MustCompile(`Minimum supported API version is ([0-9\.]+)`)
+		match := re.FindStringSubmatch(err.Error())
+		if len(match) == 2 {
+			required := match[1]
+			os.Setenv("DOCKER_API_VERSION", required)
+			cli, err = dockerclient.NewEnvClient()
+			dockerApiVersion = required
+			if err != nil {
+				return nil, err
+			}
+			_, err = cli.Info(ctx)
```

Comment:
Handle the check here as well. We should basically assert() if it worked or not as to get people complaining to us quicker.